charts/portal/templates/dispatcher/dispatcher-secret.yaml

```diff
@@ -9,8 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-{{ if .Values.dispatcher.additionalSecret }}
-{{- range $key, $val := .Values.dispatcher.additionalSecret }}
+  RABBITMQ_DEFAULT_USER: {{ .Values.rabbitmq.auth.username | b64enc | quote }}
+  {{ if .Values.dispatcher.additionalSecret }}
```

Comment:
Please remove added space to template conditional statement. Should be same as lines 12, 13 , 15 and 16.